pandas_datareader/yahoo/daily.py

```diff
@@ -97,7 +98,9 @@ def yurl(symbol):
 
     def _get_params(self, symbol):
         unix_start = int(time.mktime(self.start.timetuple()))
-        unix_end = int(time.mktime(self.end.timetuple()))
+        day_end = self.end + datetime.timedelta(days=1) \
```

Comment:
This approach is not correct.

```
import pandas_datareader as pdr
r = pdr.yahoo.daily.YahooDailyReader(['IBM'],'1/1/1960','1/1/2017 12:30:00')
r.end
Out[6]: Timestamp('2017-01-01 12:30:00')
```

Since it accepts times as well as dates you will need to create a new timestamp using the year, day and month and setting the hour to 23, minute to 59, and second to 59.